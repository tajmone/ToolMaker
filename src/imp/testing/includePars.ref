%%(test)
%%(test[0])
%%(test[1])
%%(test[2])
par11
1
par11
%%(test[2])
par21,par22
2
par21
par22
>1<
><
>2<
