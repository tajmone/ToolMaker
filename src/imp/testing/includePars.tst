%%BEGIN(test)
%%(test)
%%(test[0])
%%(test[1])
%%(test[2])
%%END(test)
%%INCLUDE(test)
%%INCLUDE(test(par11))
%%INCLUDE(test(par21, par22))
%%BEGIN(test2)
>%%(test2)<
%%END(test2)
%%SET test2(1)
%%INCLUDE(test2)
%%INCLUDE(test2())
%%INCLUDE(test2(2))
