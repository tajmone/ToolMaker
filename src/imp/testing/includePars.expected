########## includePars ##########
%%(test)
%%(test[0])
%%(test[1])
%%(test[2])
par11
1
par11
%%(test[2])
par21,par22
2
par21
par22
>1<
><
>2<


includePars.tst

    2.  %%(test)
=====>     1

  *1*   204 W : Variable "test" not set.

    3.  %%(test[0])
=====>     1

  *1*   204 W : Variable "test" not set.

    4.  %%(test[1])
=====>     1

  *1*   204 W : Variable "test" not set.

    5.  %%(test[2])
=====>     1    2

  *1*   204 W : Variable "test" not set.
  *2*   209 E : Index value "2" out of range.


1 error(s) detected.
4 warning(s) issued.

